Desktop/utilities/appdirs.h.in

```diff
@@ -44,6 +50,12 @@ class AppDirs
 	
 private:
 	static QString processPath(const QString & path);
+
+#ifdef linux
+	static QString R_HOME() {
+		return QString("@R_HOME_PATH@");
```

Comment:
I can change it to be more descriptives, but I didn't want to start tweaking macros, especially since CMake doesn't play nicely, and I have to check the length of it, but if you prefer it as a macro, I can try. I generally don't do MACRO's until I really have to.

Reply:
well, either make it a macro in caps or a function not all-caps and my confusion is gone

Reply:
I will turn it into the `AppDirs::rHomeDir()` for now.

P.S. We have so many version of `rHome` that we should probably need to look into at some point.